.github/scripts/fetch_prs.mjs

```diff
@@ -0,0 +1,54 @@
+import { Octokit } from '@octokit/core';
+import axios from 'axios';
+import moment from 'moment';
+
+const owner = 'deckhouse';
+const repo = 'virtualization';
+
+const octokit = new Octokit({ auth: process.env.RELEASE_PLEASE_TOKEN });
+
+
+async function fetchPullRequests() {
+  const pulls = await octokit.request('GET /repos/{owner}/{repo}/pulls', {
+    owner,
+    repo,
+    state: 'all'
+  });
+  return pulls.data.filter(pr => !pr.draft);
+}
+
+function generateSummary(prs) {
+  const now = moment();
+  const reviewRequired = prs.filter(pr => pr.state === 'open' && pr.requested_reviewers.length > 0);
+  const pending = reviewRequired.filter(pr => now.diff(moment(pr.created_at), 'days') <= 2);
+
+  return `
+## Daily PR Summary
+
+### PRs Requiring Review
+${reviewRequired.map(pr => `- [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()})`).join('\n') || 'No PRs requiring review.'}
+
+### PRs Pending (<=2 days)
+${pending.map(pr => `- [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()})`).join('\n') || 'No PRs pending for review (<=2 days).'}
+  `;
+}
```

Comment:
- Split recent and non-recent lists.
- Hide title for list if list is empty: reduce informational noise.
- Move recent days into constant.
- Handle "no PRs to review" case.

```suggestion
const recentDays = 2;

function generateSummary(prs) {
  const now = moment();
  const reviewRequired = prs.filter(pr => pr.state === 'open' && pr.requested_reviewers.length > 0);
  const recent = reviewRequired.filter(pr => moment().diff(moment(pr.created_at), 'days') <= recentDays);
  const lasting = reviewRequired.filter(pr => moment().diff(moment(pr.created_at), 'days') > recentDays);

  let summary = `## Daily PR Summary\n\n`;

  if reviewRequired.length == 0 {
    summary += `:tada: No review required for today\n`;
    return summary;
  }
  
  if (recent.length > 0) {
    summary += `### Recent PRs requiring review

${recent.map(pr => `- [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()})`).join('\n')}
`;
  }

  if (lasting.length > 0) {
    summary += `### PRs requiring review

${lasting.map(pr => `- [${pr.title}](${pr.html_url}) (Created: ${moment(pr.created_at).fromNow()})`).join('\n')}
`;
  }

  return summary
}
```